### ISSUE TITLE
Merge projects?

### ISSUE THREAD

**Issue by cmcaine** [ueokande/vim-vixen#258]
Hi,

I'm the project lead for Tridactyl, a similar project to vim-vixen. Our projects have similar goals and are at similar levels of development. Vim-vixen is more polished, while Tridactyl is a bit rougher but supports more features.

Are you interested in working towards merging our projects and working together at some point in the future?

I don't know how that would work, if you said yes, but I'm sure we could work something out.

All the best,
Colin

**Comment by ueokande** [comment]
@cmcaine 

Sounds great. I think that is also better for both extensions' users. But I cannot answer "ok, let's merge from now!", because there are many issues.

It expect that merging will not be well. Actually we need to decide one main repository and abandon other repository, or create a new repository from scratch. I wish to continue to develop current project now, and you may think so too.  

While your project and mine are in growing phase, it's too difficult to decide the course of the projects.  Why don't you discuss again at our project become maturity?

Thank you, very much.

P.S. I welcome issues and pull requests anytime.

**Comment by pnikosis** [comment]
I've been moving back and forth between Vim Vixen and Tridactyl, so I'd love this.

**Comment by lehasb** [comment]
Just a comment -- merging later, when projects are more mature might actually be harder, since you would have to combine two more stable code bases. If you want to work towards that, you might want to agree on some general guidelines or core pieces that are the same now, so that even if you're developing separately for the moment, you minimize the number of conflicts later.

**Comment by dkrmr** [comment]
I agree with @lehasb. Merging now would be better and less painfull than later.
Also, a unified project means more contributors.

**Comment by robsmith11** [comment]
Is anyone familiar with both projects?  Maybe it would be useful to start by identifying what features they have in common, what features are missing from each other, and what features are incompatible.

If the feature sets are largely compatible, then a deeper dive could be done to compare implementation differences and advantages/disadvantages of each approach.

**Comment by sondr3** [comment]
I agree with what the others have said, merging sooner rather than later will save you a ton of pain later on when projects start diverging. I've done this previously with a side project and I really wish I would've done it as soon as the idea came up, it was a massive time sink for what should've been a quick and easy performance and usability gain.
